tests/test_visualize.py

```diff
@@ -1,21 +1,8 @@
-import itertools
-
 import skhep_testdata
 
 import pylhe
 
 
-def test_visualize(tmpdir):
-    lhe_file = skhep_testdata.data_path("pylhe-testfile-pr29.lhe")
-    events = pylhe.read_lhe_with_attributes(lhe_file)
-
-    start_event = 1
-    stop_event = 2
-    filename = tmpdir.join(f"event{start_event}.pdf")
-    for event in itertools.islice(events, start_event, stop_event):
-        pylhe.visualize(event, filename)
```

Comment:
Any objection to keeping this in to make sure that the graphs being produced are actually valid for `graph.render` but with

```python
event = next(itertools.islice(events, 1, 2))
event.graph.render(filename="test_event1", format="pdf", cleanup=True)
```

?

Reply:
That sounds good. You're basically suggesting to test what is exemplified in the README, and that only fair.

True that test coverage is not amazing yet and needs more work.

Reply:
Now added via https://github.com/scikit-hep/pylhe/pull/131/commits/2c37af784231d618f05b0fe60d2a3909e65b5e0c.